js/src/controls/SearchControl.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+const L = require('../leaflet.js');
+const control = require('./Control.js');
+
+export class LeafletSearchControlModel extends control.LeafletControlModel {
+  defaults(){
+    return{
+      _view_name: 'LeafletSearchControlView',
+      _model_name: 'LeafletSearchControlModel',
+      url: null,
+      jsonpParam: 'json_callback',
+      propertyName: 'display_name',
+      propertyLoc: ['lat','lon'],
+      autoType: false,
```

Comment:
This attribute and some others are missing Python side

Reply:
I have added them now.